flang-rt/include/flang-rt/flang_rt/array-constructor.h

```diff
@@ -1,4 +1,4 @@
-//===-- include/FortranRuntime/Runtime/array-constructor.h ------*- C++ -*-===//
+//===-- include/flang-rt/flang_rt/array-constructor.h -----------*- C++ -*-===//
```

Comment:
Why do we need this double directory with similar names. Can the header just be in `include/flang-rt` and the CUF one in a subdirectory like before?

Reply:
I updated the summary to explain this:

The folder structure of flang-rt/include/flang-rt/flang_rt/ can be explained as follows: flang-rt (with hyphen) is the subproject's name. The second occurrence is required so #includes are prefixed with it. flang_rt is the library name (e.g. libflang_rt.a). Other library names are CufRuntime and FortranFloat128Math (which as no non-common public headers).

flang-rt follows the convention of subproject, e.g. compiler-rt
flang_rt follows the convention of Compiler-RT libraries, e.g. libclang_rt.builtins.a

Reply:
That feels strange to me. I hope we can come out with a better solution.